ngraph/core/src/op/max.cpp

```diff
@@ -67,6 +68,8 @@ bool op::v1::ReduceMax::evaluate(const HostTensorVector& outputs,
                                  const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v1_ReduceMax_evaluate);
+    NGRAPH_CHECK(validate_host_tensor_vector(inputs, 2));
```

Comment:
Looks like we don't use the second input, am I right?

Reply:
It should be probably updated to reuse `get_normalized_axes_from_tensor` helper with second input

Reply:
I had to wait for ReduceProd to introduce correctly the usage of helper to retrieve axes from host tensor. Now it makes sense to check for second input.